pkg/plugin/standalone/v1/internal/fromproto/specifier.go

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fromproto
+
+import (
+	"fmt"
+
+	"github.com/conduitio/conduit/pkg/plugin"
+	connectorv1 "go.buf.build/library/go-grpc/conduitio/conduit-plugin/connector/v1"
+)
+
+func SpecifierSpecifyResponse(in *connectorv1.Specifier_Specify_Response) (plugin.Specification, error) {
+	specMap := func(in map[string]*connectorv1.Specifier_Parameter) (map[string]plugin.Parameter, error) {
+		out := make(map[string]plugin.Parameter, len(in))
+		var err error
+		for k, v := range in {
+			out[k], err = SpecifierParameter(v)
+			if err != nil {
+				return nil, fmt.Errorf("error converting SpecifierParameter %q: %w", k, err)
+			}
+		}
+		return out, nil
+	}
+
+	sourceParams, err := specMap(in.SourceParams)
+	if err != nil {
+		return plugin.Specification{}, fmt.Errorf("error converting SourceSpec: %w", err)
+	}
+
+	destinationParams, err := specMap(in.DestinationParams)
+	if err != nil {
+		return plugin.Specification{}, fmt.Errorf("error converting DestinationSpec: %w", err)
+	}
+
+	out := plugin.Specification{
+		Name:              in.Name,
+		Summary:           in.Summary,
+		Description:       in.Description,
+		Version:           in.Version,
+		Author:            in.Author,
+		DestinationParams: destinationParams,
+		SourceParams:      sourceParams,
+	}
+	return out, nil
+}
+
+func SpecifierParameter(in *connectorv1.Specifier_Parameter) (plugin.Parameter, error) {
```

Comment:
Maybe make it unexported?

Reply:
I feel like I saw this comment before 😅   I see this (and other `fromX` and `toX` packages) as a utility package, which doesn't care about how and where it's used, it just provides the functionality, so I wouldn't hide this function as it's just another conversion function.

Reply:
Yeah, I'm sure I asked it before somewhere, but, you know "better ask, then be sorry". You're our internal Google.:D